sentry_sdk/__init__.py

```diff
@@ -19,26 +19,35 @@
     "init",
     "integrations",
     # From sentry_sdk.api
+    "add_breadcrumb",
     "capture_event",
-    "capture_message",
     "capture_exception",
-    "add_breadcrumb",
+    "capture_message",
     "configure_scope",
-    "push_scope",
+    "continue_trace",
     "flush",
+    "get_baggage",
+    "get_client",
+    "get_current_scope",
```

Comment:
It just seems like it because I sorted the list. In reality only the new APIs `get_client`, `get_current_scope`, `get_global_scope`, `get_isolation_scope`, `is_initialized`, `isolated_scope`, `new_scope`, `set_current_scope`, `set_isolation_scope` were added.

Reply:
Yes, I understand that some of the APIs were moved. My question is specifically about the `get_client`, `get_current_scope`, `get_global_scope`, `get_isolation_scope`, `set_current_scope`, and `set_isolation_scope` methods you listed as being newly added; do we really need to be adding these here in the top-level API?

Reply:
As mentioned in the sync meeting yesterday, we will have the API as minimal as possible